plugins/processors/t128_filter/README.md

```diff
@@ -0,0 +1,74 @@
+# T128 Filter Processor Plugin
+
+The `t128_filter` filters out metrics passing through it. This processor is useful if the sort of filtering you wish to accomplish which is not possible with `tagpass`/`tagdrop`. The most usual case this processor can help is if you have two tags that you want both to be a certain value before passing along the metric.
+
+[Metric selectors](docs/CONFIGURATION.md#selectors) (such as `tagpass` and `tagdrop`) can be used to only apply the filters to specific metrics.
+
+### Configuration:
+
+```toml
+[[processors.t128_filter]]
+  ## The conditions that must be met to pass a metric through. This is similar
+  ## behavior to a tagpass, but multiple tags are ANDed
+  [[processors.t128_filter.condition]]
+
```

Comment:
should there be an optional `name` field for the `condition`? In debug mode, we can then log saying which condition was met. By default the names would be condition1, condition2 etc using their relative positin in the config.

Reply:
I like that. Trying to figure out how to implement that with the structure that I have without making things gross.